modules/runtime/nuxeo-runtime/src/main/resources/OSGI-INF/ContributionPersistence.xml

```diff
@@ -19,7 +19,7 @@
     <provide interface="org.nuxeo.runtime.model.persistence.ContributionPersistenceManager" />
   </service>
 
-  <extension-point name="listeners">
+  <extension-point name="storage">
```

Comment:
The previous code did a fallback in case there is no contribution (which is still the case even in tests): registry retrieval threw an error so that's how I saw it...